datm/cime_config/testdefs/testlist_datm.xml

```diff
@@ -95,6 +98,7 @@
     </machines>
     <options>
       <option name="wallclock"> 00:10:00 </option>
+      <option name="comment">Test historical CRUJRA2024 forcing in a short smoke test to make sure it works</option>
```

Comment:
Change the compset to be HIST, rather than SSP585.

Reply:
Will do. (I did SSP585 to replicate the GSWP3 test.)